### ISSUE TITLE
[docs] update Menu/About text

### ISSUE THREAD

**Issue by LeonNicholls** [teia-community/teia-ui#1]
The text has bad grammar and was written for HEN goals. Consider updating it for Teia community goals.

**Comment by merchantcoppola** [comment]
included in this issue: https://github.com/teia-community/teia-ui/issues/22

**Comment by merchantcoppola** [comment]
i will work on a new copy text for the about page

**Comment by merchantcoppola** [comment]
will add the text once it is finished to the copytext collection issue here: https://github.com/teia-community/teia-ui/issues/34

**Comment by melMass** [comment]
Closing in favor of #34
